Physics/CdrPlasma/PlasmaModels/CdrPlasmaJSON/CD_CdrPlasmaJSON.cpp

```diff
@@ -2236,7 +2236,7 @@ void CdrPlasmaJSON::parseElectrodeReactions() {
 	this->sanctifySurfaceReaction(curReactants, curProducts, reaction);
 
 	// This is the reaction index for the current index. The reaction we are currently
-	// dealing with is put in m_plasmaReactions[reactionIdex]. 
+	// dealing with is put in m_electrodeReactions[reactionIndex]. 
```

Comment:
Is my version correct? If yes: I'll fix it for dielectric as well, if no: I'll change it back